.mergify.yml

```diff
@@ -0,0 +1,37 @@
+pull_request_rules:
+  - name: automatic merge
+    conditions:
+      - and: &base_checks
+        - base=master
+        - -label~=^acceptance-tests-needed|not-ready
+        - -body~=PLACEHOLDER
+        - "status-success~=static tests.*"
+        - "status-success~=unit tests.*"
+        - "status-success~=compile tests.*"
+        - "#check-failure=0"
+        - "#check-pending=0"
+        - linear-history
+      - and:
+        - "#approved-reviews-by>=2"
+        - "#changes-requested-reviews-by=0"
+        # https://doc.mergify.io/examples.html#require-all-requested-reviews-to-be-approved
+        - "#review-requested=0"
+        - "body~=Verification run: http"
+    actions: &merge
+      merge:
+        method: merge
+  - name: automatic merge on special label
+    conditions:
+      - and: *base_checks
+      - and:
+        # mergify config checks needs at least two rules in "and" so we repeat
+        # one from the base checks
+        - base=master
+        - "label=merge-fast"
+    actions: *merge
+  - name: ask to resolve conflict
+    conditions:
+      - conflict
+    actions:
+      comment:
+        message: This pull request is now in conflicts. Could you fix it? 🙏
```

Comment:
I always found this text weird. The emoji doesn't really fit, or do I just not understand it?

Reply:
The common meaning is "please", see https://emojipedia.org/folded-hands/ . I would prefer to keep the very same text we use for other repos in https://github.com/os-autoinst

Reply:
I'd prefer to write: Could you *please* fix it?, but if you think emoji is important, I don't have strong opinion. I'd just try to avoid utf8 symbols if possible (not long time ago copyright symbol was removed).

Reply:
Sure, I can change it. I just don't see why we should treat os-autoinst-distri-opensuse any different from [os-autoinst](https://github.com/os-autoinst/os-autoinst/blob/master/.mergify.yml#L39), [openQA](https://github.com/os-autoinst/openQA/blob/master/.mergify.yml#L40), [scripts](https://github.com/os-autoinst/scripts/blob/master/.mergify.yml#L31), [openqa-review](https://github.com/os-autoinst/openqa_review/blob/master/.mergify.yml#L29), [os-autoinst-common](https://github.com/os-autoinst/os-autoinst-common/blob/master/.mergify.yml#L26)

Reply:
I also vote for keeping that emoji as it makes the workflow a bit more colourful. and even screen readers nowadays recognise those.

Reply:
so let's keep it in this way, same as we have for other projects